### DISCUSSION TITLE
How to associate Activity model with other model e.g TrackInfo

### DISCUSSION THREAD

**Discussion by sharoonamjid** [spatie/laravel-activitylog#974] [Q&A]
I'm thinking of a solution to save information of user e.g ip_address, browser, device_name etc whenever Activity Log is being saved. I'm thinking of making another table called track_infos where I'll have foreign id of activity_log_id. But I'm not sure How to define relationship between Activity model and TrackInfo Model.

all I want is to achieve:

Activity::with('track_infos')->get();

Looking forward to hear.

**Comment by Gummibeer** [comment]
https://github.com/spatie/laravel-activitylog/blob/a23de07a6160897559f7f9ac14fa98d2676d450a/config/activitylog.php#L33-L38
https://laravel.com/docs/8.x/eloquent-relationships
https://laravel.com/docs/8.x/eloquent#events
